src/pages/clients/edit/components/AdditionalInfo.tsx

```diff
@@ -46,14 +43,7 @@ export function AdditionalInfo({ client, errors, setClient }: Props) {
   const currencies = useCurrencies();
   const languages = useLanguages();
 
-  const { data: paymentTerms } = useQuery({
-    queryKey: ['/api/v1/payment_terms'],
-    queryFn: () =>
-      request('GET', endpoint('/api/v1/payment_terms')).then(
-        (response: AxiosResponse<GenericManyResponse<PaymentTerm>>) =>
-          response.data.data
-      ),
-  });
+  const { data: paymentTermsResponse } = usePaymentTermsQuery({});
```

Comment:
@beganovich The `usePaymentTermsQuery` hook is used on a table where the `meta_data` is required. Therefore, the `response` is not refactored, and `paymentTermsResponse.data.data` is used in the logic.